spec/locales-currencies-tz.html

```diff
@@ -248,11 +248,11 @@ <h1>IsSanctionedSimpleUnitIdentifier ( _unitIdentifier_ )</h1>
       </emu-alg>
 
       <emu-table id="table-sanctioned-simple-unit-identifiers">
-        <emu-caption>Simple units sanctioned for use in ECMAScript</emu-caption>
+        <emu-caption>Single units sanctioned for use in ECMAScript</emu-caption>
```

Comment:
As described in [UTS #35](https://www.unicode.org/reports/tr35/tr35-general.html#Unit_Identifiers), prefixed unit identifiers such as "centimeter" and "gigabit" are not _simple_. I've updated the text, but I'm thinking I should also rename the algorithm and replace the table id. Thoughts, @tc39/ecma402-editors?

Reply:
Yeah, the terms "single" and "simple" were hammered down in UTS 35 and now ECMA-402 has bitrot.  I think we should use the new UTS 35 terminology ("single" rather than "simple").